source/install/self-managed-changelog.md

```diff
@@ -5,11 +5,121 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v5.37. 
 
 Latest Mattermost Releases:
+- [Release v6.7 - Feature Release](#release-v6-7-feature-release)
 - [Release v6.6 - Feature Release](#release-v6-6-feature-release)
 - [Release v6.5 - Feature Release](#release-v6-5-feature-release)
 - [Release v6.4 - Feature Release](#release-v6-4-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
-- [Release v5.37 - Extended Support Release](#release-v5-37-extended-support-release)
+
+## Release v6.7 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2022-05-16**
+
+### Compatibility
+ - Updated Chrome recommended minimum version to v100+.
+
+### Important Upgrade Notes
+ - New schema changes were introduced in the form of a new index. The following summarizes the test results measuring how long it took for the database queries to run with these schema changes:
+    - MySQL 7M Posts - ~17s (Instance: db.r5.xlarge)
+    - MySQL 9M Posts - 2min 12s (Instance: db.r5.large)
+    - Postgres 7M Posts - ~9s  (Instance: db.r5.xlarge)
+ - For customers wanting a zero downtime upgrade, they are welcome to apply this index prior to doing the upgrade. This is fully backwards compatible and will not acquire any table lock or affect any existing operations on the table. Run the following to apply this index:
```

Comment:
```suggestion
 - For customers wanting a zero downtime upgrade, they are welcome to apply this index prior to doing the upgrade. Else, the queries will run during the upgrade process. This is fully backwards compatible and will not require any table lock or affect any existing operations on the table. Run the following to apply this index:
```

Reply:
^assuming this is accurate

Reply:
Hmm .. any reason you replaced `acquire` with `require`? The meaning doesn't change very much but was just curious.

And just to clarify, the not acquiring table lock part will _only_ work if you run the queries before. The index application in the postgres case is not lock-free when done from the application. Only the MySQL one is. That's why I avoided the confusion and said only externally ran queries will be lock-free.

After you inserted the "Else, the queries will run during the upgrade process. " sentence, it might start to mean that queries ran during the upgrade process will also be lock-free.

Reply:
Thanks @agnivade! How is this?

```
For customers wanting a zero downtime upgrade, they are encouraged to apply this index prior to doing the upgrade. This is fully backwards compatible and will not acquire any table lock or affect any existing operations on the table when run manually. Else, the queries will run during the upgrade process and will lock the table in non-MySQL environments. Run the following to apply this index:
```

I used the term `require` to emphasize otherwise it's required. But it sounds like it's not quite accurate, so adjusted it back

Reply:
Yep, looks good!